assets/javascripts/discourse/components/ai-llms-list-editor.gjs

```diff
@@ -39,60 +90,118 @@ export default class AiLlmsListEditor extends Component {
-                    <LinkTo
-                      @route="adminPlugins.show.discourse-ai-llms.show"
-                      current-when="true"
-                      class="btn btn-text btn-small"
-                      @model={{llm}}
-                    >{{i18n "discourse_ai.llms.edit"}}</LinkTo>
-                  </td>
+        {{#if this.hasLlmElements}}
+          <section class="ai-llms-list-editor__configured">
+            <div class="admin-page-subheader">
+              <div class="admin-page-subheader__title-row">
+                <h2 class="admin-page-subheader__title">
+                  {{i18n "discourse_ai.llms.configured.title"}}
+                </h2>
+              </div>
+            </div>
+            <table>
+              <thead>
+                <tr>
+                  <th>{{i18n "discourse_ai.llms.display_name"}}</th>
+                  <th>{{i18n "discourse_ai.llms.provider"}}</th>
+                  <th>{{i18n "discourse_ai.llms.enabled_chat_bot"}}</th>
+                  <th></th>
                 </tr>
-              {{/each}}
-            </tbody>
-          </table>
+              </thead>
+              <tbody>
+                {{#each @llms as |llm|}}
+                  <tr data-persona-id={{llm.id}} class="ai-llm-list__row">
+                    <td class="column-name">
+                      <h3>{{llm.display_name}}</h3>
+                      <p>
+                        {{i18n
+                          (concat
+                            "discourse_ai.llms.model_description."
+                            llm.provider
+                            "-"
+                            llm.name
+                          )
+                        }}
+                      </p>
+                    </td>
+                    <td>
+                      {{i18n
+                        (concat "discourse_ai.llms.providers." llm.provider)
+                      }}
+                    </td>
+                    <td>
+                      <DToggleSwitch
+                        @state={{llm.enabled_chat_bot}}
+                        {{on "click" (fn this.toggleEnabledChatBot llm)}}
+                      />
+                    </td>
+                    <td class="column-edit">
+                      <LinkTo
+                        @route="adminPlugins.show.discourse-ai-llms.show"
+                        class="btn btn-default"
+                        @model={{or llm.id llm.llm.id}}
+                      >
+                        {{icon "wrench"}}
+                        <div class="d-button-label">
+                          {{i18n "discourse_ai.llms.edit"}}
+                        </div>
+                      </LinkTo>
+                    </td>
+                  </tr>
+                {{/each}}
+              </tbody>
+            </table>
+          </section>
         {{/if}}
+        <section class="ai-llms-list-editor__templates">
+          <div class="admin-page-subheader">
```

Comment:
Please use the actual `AdminPageSubheader` component from core here, here is an example:

https://github.com/discourse/discourse/blob/c5a024f8dfcd0611f3661b59b0b7c8d5bc72fac8/app/assets/javascripts/admin/addon/components/admin-config-areas/flags.gjs#L72-L83

Also /t/129128 internally has docs